docs/src/jsx/MouseEvents.js

```diff
@@ -0,0 +1,131 @@
+//  Copyright (c) 2018-present, GM Cruise LLC
+//
+//  This source code is licensed under the Apache License, Version 2.0,
+//  found in the LICENSE file in the root directory of this source tree.
+//  You may not use this file except in compliance with the License.
+
+// #BEGIN EXAMPLE
+import last from "lodash/last";
+import remove from "lodash/remove";
+import sample from "lodash/sample";
+import React, { useState } from "react";
+import seedrandom from "seedrandom";
+
+import Worldview, { Cubes } from "regl-worldview";
+
+// #BEGIN EDITABLE
+function Example() {
+  const rng = seedrandom();
+  const randomCubeFacts = [
+    "A cube is a three dimensional shape",
+    "A cube has 6 square faces.",
+    "A cube has the largest volume of all cuboids with a certain surface area",
+    "All the matter that makes up the human race could fit in a sugar cube",
+    "There are 43,252,003,274,489,856,000 possible permutations of the Rubik's Cube",
+    "Ice Cube was the only member of NWA not born in Compton",
+    "Wombats have cube-shaped poop",
+  ];
+
+  const randomMouseCursors = ["progress", "wait", "crosshair", "text", "move", "grab", "ew-resize", "ns-resize"];
+
+  const [cubes, setCubes] = useState([createCube(0)]);
+  const [cubeDetails, setCubeDetails] = useState({});
+  const [cursor, setCursor] = useState("auto");
+
+  function cube(range, id) {
+    const marker = {
+      id,
+      pose: {
+        orientation: { x: 0.038269, y: -0.01677, z: -0.8394, w: 0.541905 },
```

Comment:
we are trying to reduce magic number, would the orientation {x: 0, y: 0, z:0, w: 1} work?